docs/source/demand_side_agent.rst

```diff
@@ -32,18 +53,20 @@ Each component (e.g., Electrolyser, Heat Pump, DRI Plant) is modeled in detail w
 - Operating costs
 - Load flexibility
 
+For full references see :mod:`assume.units.dst_components`.
+
 Bidding and Optimization
 ------------------------
 
 Once the DSU is defined, it can participate in the electricity market by optimizing its operation to either minimize costs or maximize flexibility. The bidding strategy depends on the agent type and market conditions.
 
 Industrial Agent: Steel Plant
-=============================
+-----------------------------
```

Comment:
most of this is still Attributes which look like they duplicate the docstrings.. But well, it was the case before as well..